README.md

```diff
@@ -11,3 +11,11 @@ The app (currently beta! All users globally still share the same session) is ava
 2. Install pre-commit: `uv run pre-commit install`
 3. To run pre-commit manually run: `uv run pre-commit run --all-files`
 4. To run app locally: `uv run streamlit run main.py`
+
+### Dependency Management
+
+This project uses `uv` for dependency management:
+
+- **uv.lock**: Lockfile with exact versions for all platforms. Committed to version control.
```

Comment:
Please link to files instead of just printing them bold

Reply:
done